tools/cockpit.spec

```diff
@@ -175,67 +200,121 @@ install -p -m 644 cockpit.pp %{buildroot}%{_datadir}/selinux/targeted/
 %ifnarch x86_64
 rm -rf %{buildroot}/%{_datadir}/%{name}/docker
 %endif
+%if 0%{?suse_version}
+%suse_update_desktop_file -u -r -G 'Cockpit Server Manager' %{name} System Utility
+mv %{buildroot}%{_datadir}/doc/%{name} %{buildroot}%{_docdir}/%{name}/html
+%endif
 
 %files
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %{_docdir}/%{name}/AUTHORS
 %{_docdir}/%{name}/COPYING
 %{_docdir}/%{name}/README.md
+%if 0%{?suse_version}
+%dir %{_datadir}/%{name}
+%endif
 %{_datadir}/appdata
 %{_datadir}/applications
 %{_datadir}/pixmaps
 
 %files bridge
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %doc %{_mandir}/man1/cockpit-bridge.1.gz
 %{_bindir}/cockpit-bridge
 %attr(4755, -, -) %{_libexecdir}/cockpit-polkit
+%if 0%{?suse_version}
+%dir %{_libdir}/security
+%endif
 %{_libdir}/security/pam_reauthorize.so
 
 %files daemon
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %doc %{_mandir}/man8/cockpitd.8.gz
 %{_datadir}/dbus-1/services/com.redhat.Cockpit.service
 %{_libexecdir}/cockpitd
 
 %files doc
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
+%exclude %{_docdir}/%{name}/AUTHORS
+%exclude %{_docdir}/%{name}/COPYING
+%exclude %{_docdir}/%{name}/README.md
 %{_docdir}/%{name}
 
 %files shell
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %{_datadir}/%{name}/base
 %{_datadir}/%{name}/shell
 %{_datadir}/%{name}/playground
 %{_datadir}/%{name}/server-systemd
 
+%if !0%{?suse_version}
 %post shell
 # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1185749
 ( cd /var/lib/pcp/pmns && ./Rebuild -du )
+%endif
 
 %files ws
+%if 0%{?suse_version}
+%defattr(-,root,root)
+%endif
 %doc %{_mandir}/man5/cockpit.conf.5.gz
 %doc %{_mandir}/man8/cockpit-ws.8.gz
 %config(noreplace) %{_sysconfdir}/%{name}
 %config(noreplace) %{_sysconfdir}/pam.d/cockpit
 %{_unitdir}/cockpit.service
 %{_unitdir}/cockpit.socket
+%if 0%{?suse_version}
+%dir %{_prefix}/lib/firewalld
+%dir %{_prefix}/lib/firewalld/services
+%endif
 %{_prefix}/lib/firewalld/services/cockpit.xml
 %{_sbindir}/remotectl
 %{_libexecdir}/cockpit-ws
 %attr(4750, root, cockpit-ws) %{_libexecdir}/cockpit-session
+%if !0%{?suse_version}
 %attr(775, -, wheel) %{_sharedstatedir}/%{name}
+%endif
 %{_datadir}/%{name}/static
 
 %pre ws
+%if 0%{?suse_version}
+%service_add_pre %{name}.service
+%endif
 getent group cockpit-ws >/dev/null || groupadd -r cockpit-ws
 getent passwd cockpit-ws >/dev/null || useradd -r -g cockpit-ws -d / -s /sbin/nologin -c "User for cockpit-ws" cockpit-ws
 
 %post ws
+%if 0%{?suse_version}
+%service_add_post %{name}.service
```

Comment:
Why does this use the cockpit.service while the Fedora version refers to cockpit.socket? The intent is that cockpit be socket activated. The cockpit.socket is the thing that's enabled and started, and the service not manually enabled or started. At least in the default configuration.